web/templates/web/header.html

```diff
@@ -6,78 +6,92 @@
+                <i class="dropdown icon nav-item-text" tabindex{# For disabling focus on click #}></i>
+                <div class="bubble-background"></div>
+
                 <div class="menu transition hidden">
                     {% if user|has_any_permissions or user.is_superuser %}
-                        <a class="item" href="{% url 'adminpanel' %}">
-                            {% trans "Administration" %}
+                        <a class="dropdown-item item" href="{% url 'adminpanel' %}">
+                            <div class="dropdown-item-text">{% trans "Administration" %}</div>
                         </a>
                     {% endif %}
-                    <a class="item" href="{% url 'profile' %}">
-                        {% trans "Profile" %}
+                    <a class="dropdown-item item" href="{% url 'profile' %}">
+                        <div class="dropdown-item-text">{% trans "Profile" %}</div>
                     </a>
-                    <a class="item" href="{% url 'logout' %}">
-                        {% trans "Logout" %}
+                    <a class="dropdown-item item" href="{% url 'logout' %}">
+                        <div class="dropdown-item-text">{% trans "Logout" %}</div>
                     </a>
                 </div>
+
             </div>
         {% else %}
-            <a class="item" href="{% url 'login' %}">
-                {% trans "Login" %}
+            <a class="nav-item item" href="{% url 'login' %}">
+                <div class="nav-item-text">{% trans "Login" %}</div>
+                <div class="bubble-background"></div>
             </a>
         {% endif %}
-        <!--div class="right menu">
-            <div class="item">
-                <div class="ui icon input">
-                    <input type="text" placeholder="Search...">
-                    <i class="search link icon"></i>
+
+        <div class="lang-link ui inverted secondary menu">
+            <div class="ui language link item">
+                <div class="text">
+                    {% if LANGUAGE_CODE == "en" %}
+                        Norsk
+                    {% else %}
+                        English
+                    {% endif %}
                 </div>
+                <i class="globe icon text"></i>
             </div>
-            <a class="ui item">
-                Logout
-            </a>
-        </div-->
-        <a class="item lang-link" href="">
-            {% if LANGUAGE_CODE == "en" %}
-                Norsk
-            {% else %}
-                English
-            {% endif %}
-        </a>
+        </div>
+
     </div>
-</div>
+</header>
 
-{# This form is hidden, but is submitted through JQuery #}
-<form class="lang-form" action="{% url 'set_language' %}" method="POST">
+{# This form is hidden, but is submitted through jQuery #}
+<form id="lang-form" action="{% url 'set_language' %}" method="POST" style="display:none">
```

Comment:
Just moved `display: none` from the stylesheet, because I felt it didn't really belong there, and to make it more explicit. Should I remove the [submit button](https://github.com/MAKENTNU/web/blob/wip/navbar/web/templates/web/header.html#L103) instead?

Reply:
You can just remove the submit button and the `style="display: none"`. The submit button is not used (it is hidden) and since all of the input fields are hidden, nothing will be displayed.